### PR TITLE
chore: expand browsers component spec

### PR DESCRIPTION
**Pull request by bahmutov**
Internal tests
- update cypress-react-unit-test to latest
- expand the Browsers drop-down list to look the same way as in desktop-gui
- add test to make sure the select browser method is called
- [x] can we use the real Project Model instance with the component?

Screen Shot 2020-08-14 at 3 19 56 PM

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -13789,6 +13781,12 @@ has-ansi@^2.0.0:
   dependencies:
     ansi-regex "^2.0.0"
 
+has-binary2@cypress-io/has-binary#8580a33df21e8b36a43f57872a82c60829636a92:
+  version "1.0.2"
+  resolved "https://codeload.github.com/cypress-io/has-binary/tar.gz/8580a33df21e8b36a43f57872a82c60829636a92"
+  dependencies:
+    isarray "2.0.1"
+
```
Comment (jennifer-shehane) [inline yarn.lock:13789]: @bahmutov Why is this included in your lockfile? Get nervous seeing `has-binary2` pop up again.

Comment (flotwig) [reply yarn.lock:13789]: :scream: 

@bahmutov I believe the final fix for this is to `rm -rf ~/.cache/yarn`

I am still rattled that `yarn` can non-deterministically generate a lockfile based on whatever random dependencies you have cached locally... that seems like a problem...

**Comment by bahmutov** [comment]
Nah, I go back to yarn.lock and manually modify the has-binary2. I know, but I don’t trust the fresh install at all

**Review by flotwig — DISMISSED** [review]
@bahmutov you have some weirdness going on with your local yarn, I took the following steps to cleanly update the yarn.lock on this branch:

1. `git checkout origin/develop -- yarn.lock`
2. `yarn docker`
3. `git clean -xfd`
4. `yarn`

Doing it this way, with a clean yarn cache, I was able to get a much smaller yarn.lock diff...

**Comment by bahmutov** [comment]
@flotwig I appreciate this - can you add your suggestion to the development markdown file, please? I would hate to lose such a great monorepo tip

**Comment by bahmutov** [comment]
you broke the typescript lint again @flotwig !!!! (screaming into the pillow)

**Comment by flotwig** [comment]
:'( well, better here than in `develop` when someone eventually canonically updates yarn.lock

**Comment by bahmutov** [comment]
But it was working ;)

**Review by flotwig — APPROVED** [review]
